mathjax3-ts/util/Options.ts

```diff
@@ -149,8 +163,17 @@ export function insert(dst: OptionList, src: OptionList, warn: boolean = true) {
         let sval = src[key], dval = dst[key];
         if (isObject(sval) && dval !== null &&
             (typeof dval === 'object' || typeof dval === 'function')) {
-            if (Array.isArray(dval) && Array.isArray(sval[APPEND]) && keys(sval).length === 1) {
-                dval.push(...sval[APPEND]);
+            const ids = keys(sval);
+            if (Array.isArray(dval) &&
+                ((ids.length === 1 && (ids[0] === APPEND || ids[0] === REMOVE) && Array.isArray(sval[ids[0]])) ||
+                  (ids.length === 2 && ids.sort().join(',') === APPEND + ',' + REMOVE &&
+                   Array.isArray(sval[APPEND]) && Array.isArray(sval[REMOVE])))) {
```

Comment:
Could you space out and comment these cases.

Reply:
Done.